subjects/zappy/README.md

```diff
@@ -0,0 +1,351 @@
+
+- The player will start the enchantment and the evolution will start.
+- It is not necessary that all the players are on the same team, but they need to have the same levels.
+- All players participating in the enchantment will reach a higher level.
+
+Here are the combinations of resources and players you need for the enchantment and evolution:
+
+| Level | Number of Players | Jade | Peridot | Amber | Amethyst | Garnet | Ammolite |
+| :---: | :---------------: | :--: | :-----: | :---: | :------: | :----: | :------: |
+|  1-2  |         1         |  1   |    0    |   0   |    0     |   0    |    0     |
+|  2-3  |         2         |  1   |    1    |   1   |    0     |   0    |    0     |
+|  3-4  |         2         |  2   |    0    |   1   |    0     |   2    |    0     |
+|  4-5  |         4         |  1   |    1    |   2   |    0     |   1    |    0     |
+|  5-6  |         4         |  1   |    2    |   1   |    3     |   0    |    0     |
+|  6-7  |         6         |  1   |    2    |   3   |    0     |   1    |    0     |
+|  7-8  |         6         |  2   |    2    |   2   |    2     |   2    |    1     |
+
+### Vision
+
+- The player's view of the field is limited! In order to improve your vision you will have to level up.
+
+With each evolution, your vision goes up one unit of measurement ahead and one on either side of a new line.
+
+Here is the example for the first 3 levels (our player is at 0):
+
+```console
+
+              ---- ---- ----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 1
+
+         ---- ---- ---- ---- ----
+        | 4  |  5 |  6 |  7 |  8 |
+         ---- ---- ---- ---- ----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 2
+
+ ----- ----- ----- ----- ----- ----- -----
+|  9  | 10  | 11  | 12  | 13  | 14  | 15  |
+ ----- ----- ----- ----- ----- ----- -----
+      |  4  |  5  |  6  |  7  |  8  |
+       ----- ----- ----- ----- -----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 3
+
+```
+
+For the player to know what's around him, the `client` sends the command `SEE` and the server responds the following chain of characters (for level 1):
+
+```console
+{content-square-0, content-square-1, content-square-2, content-square-3}
+```
+
+Our player cannot see himself and if there is more than 1 object in a square, they are all indicated and separated by a space:
+
+```console
+              -------- -------- --------
+             | Amber  | Garnet |        |
+             | player | Garnet |        |
+              -------- -------- --------
+                      |        |
+                      |  food  |
+                       --------
+                        Level 1
+
+```
+
+```console
+{food, player Amber, Garnet Garnet, }
```

Comment:
Here some items are all lower case and some others aren't, maybe it's better to have everything lowercase.
Also do we want to have the last `,` at the end of the list?

Reply:
done

Reply:
Maybe we should also remove the `,` and the space before the closing bracket, since in other places like inventory it seems we don't leave trailing `,`

Reply:
they are there because there is an empty square that the player sees and that way he knows it is empty.
in the other readmes the have the same info as well.